ccache.h

```diff
@@ -74,6 +74,9 @@ enum stats {
 #define str_eq(s1, s2) (strcmp((s1), (s2)) == 0)
 #define str_startswith(s, p) (strncmp((s), (p), strlen((p))) == 0)
 
+/* buffer size for IO operations */
+#define READ_BUF_SZ (1<<18)
```

Comment:
Maybe use different constants, for hash (old 16K) and copy (old 10K) ?
And maybe also configurable, for different systems / target platforms...